.github/workflows/validate.yml

```diff
@@ -22,6 +22,7 @@ jobs:
           name: logs
           path: |
             tests/test-suite.log
+            tests/pylint/runpylint*.log
```

Comment:
Please add also ``tests/nosetests.log``.

Reply:
nosetests output is already contained in test-suite.log, so that seemed redundant to me. But it's also tiny, so I updated the commit to collect it.

Reply:
It is but it gives you nice overview when there is plenty of pylint errors too.